### PR TITLE
Updated channel and documentation to Rubocop v1.4.1

### PR DESCRIPTION
**Pull request by DanielWright**
The versioning scheme change to `1.x` was kinda disruptive! A handful of dependencies were unable to resolve following the update. These have been patched, but in two cases the associated PRs have not yet been merged and released to RubyGems. Merging this should probably wait for that to happen.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,15 +6,17 @@ gem "activesupport", require: false
 gem "mry", require: false
 gem "parser"
 gem "pry", require: false
-gem "rubocop", "0.92.0", require: false
-gem "rubocop-i18n", require: false
-gem "rubocop-migrations", require: false
+gem "rubocop", "1.4.1", require: false
+gem "rubocop-i18n", require: false,
+  github: "mvz/rubocop-i18n", ref: "5e824a47a88732830a8b60e46697bbb0a2795c19"
```
Comment (DanielWright) [inline Gemfile:11]:
```suggestion
gem "rubocop-i18n", require: false
```
This change should be applied when https://github.com/puppetlabs/rubocop-i18n/pull/46 is merged and released.

File: Gemfile [hunk 2]
```diff
@@ -6,15 +6,17 @@ gem "activesupport", require: false
...
+gem "rubocop-migrations", require: false,
+  github: "DanielWright/rubocop-migrations", branch: "dw/nov24/rubocop-1.x-support"
```
Comment (DanielWright) [inline Gemfile:13]:
```suggestion
gem "rubocop-migrations", require: false
```
This change should be applied when https://github.com/redbooth/rubocop-migrations/pull/3 is merged and released.

**Review by DanielWright — COMMENTED** [review]
This review stages the changes needed when the associated dependency PRs are merged and released.

File: Gemfile (hunk 1 above)
Comment (ThomasKoppensteiner) [reply Gemfile:11]: @DanielWright wouldn't this PR be merged faster, if you stick to @mvz s [branch](https://github.com/mvz/rubocop-i18n/tree/upgrade-to-rubocop-1-0) for now?
I suggest to move this change into another pull request.

**Comment by filipesperandio** [comment]
@DanielWright the channel branch was causing some confusion to other customers, so we delete it for now. I think by doing so we closed this PR automatically. Please, feel free to re-open it (PR and branch) when you feel we are in a better place to ship this version.

**Pull request closed without merging**